File: Data Structures & Algorithms/remove-duplicates-from-sorted-array/submission-0.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int insertidx = 1;
        for(int i=1; i<nums.size(); i++){
            if(nums[i-1] != nums[i]){
                nums[insertidx] = nums[i];
                insertidx++;
            }
        }

        return insertidx;
    }
};

File: Data Structures & Algorithms/sort-array-by-increasing-frequency/submission-0.py
class Solution:
    def frequencySort(self, nums: List[int]) -> List[int]:
        mp = {}

        for n in nums:
            mp[n] = mp.get(n,0)+1

        res = []

        arr = sorted(mp, key = lambda x: (mp[x],-x))

        for n in arr:
            for __ in range(mp[n]):
                res.append(n)
        return res